resources/install/debian/postinst

```diff
@@ -152,6 +152,12 @@ case "$1" in
         # OpenVZ containers do not typically allow permission to modify the
         # kernel with sysctl config values.
         sysctl --system || true
+
+        # Make sure the service is enabled, on upgrade from old to new bridge
+        # we see the service as masked (failing to stop on remove)
+        if [ -d /run/systemd/system ]; then
+            systemctl unmask jitsi-videobridge
```

Comment:
should this be `jitsi-videobridge2`?

Reply:
This was removed in later commits.

Reply:
It is automatically handled by debhelper.

Reply:
I was trying to find out what was wrong with upgrade till I found that debhelper was not adding the stuff for systemd because the unit file was still with the old name, but it needs to match the package. Upgrades now works, but clean install still have some problems I'm looking into. I closed the PR so we do not do milion of PR tests while I'm experimenting in my fork branch.